interface-definitions/load-balancing-haproxy.xml.in

```diff
@@ -0,0 +1,200 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="load-balancing">
+    <children>
+      <node name="reverse-proxy" owner="${vyos_conf_scripts_dir}/load-balancing-haproxy.py">
+        <properties>
+          <help>Configure reverse-proxy load-balancing</help>
+        </properties>
+        <children>
+          <tagNode name="server">
+            <properties>
+              <help>Frontend name</help>
+            </properties>
+            <children>
+              <leafNode name="backend">
+                <properties>
+                  <help>Backend member</help>
+                  <valueHelp>
+                    <format>txt</format>
+                    <description>Backend name</description>
+                  </valueHelp>
+                  <completionHelp>
+                    <path>load-balancing app backend</path>
+                  </completionHelp>
+                  <multi/>
+                </properties>
+              </leafNode>
+              #include <include/listen-address.xml.i>
+              #include <include/haproxy/mode.xml.i>
+              #include <include/port-number.xml.i>
+            </children>
+          </tagNode>
+          <tagNode name="backend">
+            <properties>
+              <help>Backend name</help>
```

Comment:
Same constaint for alphanumeric as above

Reply:
Done!